packages/grid/_modules_/grid/components/ErrorOverlay.tsx

```diff
@@ -1,12 +1,21 @@
 import * as React from 'react';
 import { GridApiContext } from './GridApiContext';
-import { GridOverlay } from './containers/GridOverlay';
+import { GridOverlay, GridOverlayProps } from './containers/GridOverlay';
 
-export interface ErrorOverlayProps {
+export interface ErrorOverlayProps extends GridOverlayProps {
   message?: string;
 }
-export function ErrorOverlay({ message }: ErrorOverlayProps) {
-  const apiRef = React.useContext(GridApiContext);
-  const defaultLabel = apiRef!.current.getLocaleText('errorOverlayDefaultLabel');
-  return <GridOverlay>{message || defaultLabel}</GridOverlay>;
-}
+
+export const ErrorOverlay = React.forwardRef<HTMLDivElement, ErrorOverlayProps>(
+  function ErrorOverlay(props: ErrorOverlayProps, ref) {
+    const { message, className, ...other } = props;
+    const apiRef = React.useContext(GridApiContext);
+    const defaultLabel = apiRef!.current.getLocaleText('errorOverlayDefaultLabel');
+
+    return (
+      <GridOverlay ref={ref} className={className} {...other}>
```

Comment:
Why handle the `className` differently than the other spreaded props? Unless we add a global class name to three ErrorOverlay, it doesn't seem to be relevant to explicitly bind the prop.

Reply:
Now, if we consider that we will add global class names to all these individual comments as in the core package, it helps

Reply:
No particular reason. Saw it in a couple of other places. Now that I look at it it doesn't need to be handled differently. I don't think a dedicated class names to these components.

Reply:
Updated